public/app/redux/useReduxQuerySync.ts

```diff
@@ -99,6 +99,8 @@ function setupReduxQuerySync() {
   });
 }
 
+const unsubscribe = setupReduxQuerySync();
+
 export function useReduxQuerySync() {
-  useEffect(setupReduxQuerySync, []);
+  useEffect(() => unsubscribe, []);
```

Comment:
Won't this cause it to be unsubscribed on the first unmount, but never `setupReduxQuerySync` again?

Reply:
In the case of the app, it's never unmounted right?